src/ui/interface.js

```diff
@@ -68,6 +68,21 @@ module.exports = function Interface(options) {
       return true;
     });
 
+    $("#sample-image__select").click(function(e){
+      e.stopPropagation();
+      const img = (e.target.classList.contains('rfi')) ? document.getElementById('rfi') : document.getElementById('bfi');
+      fetch(img.src)
+        .then(res => res.blob())
+        .then(blob => {
+          const file = new File([blob], img.src, blob)
+          const fileInput = document.querySelector('input[type="file"]');
+          const dataTransfer = new DataTransfer();
+          dataTransfer.items.add(file);
```

Comment:
Wow i really like this way of doing it. It literally selects the image, same as someone using it! That will mean that we don't have to support (or think about) 2 different means of adding images. Very cool.